config/rwahs/components/providence

```diff
@@ -28,7 +28,9 @@ function postDeploy {
     localSettingsRoot="${2}"
 
     ln -s "${localSettingsRoot}/setup.php" "${targetRoot}/${targetParent}/current/setup.php"
-    ln -s "${localSettingsRoot}/media/collectiveaccess" "${targetRoot}/${targetParent}/current/media/collectiveaccess"
+    ln -s "${localSettingsRoot}/../storage/media/collectiveaccess" "${targetRoot}/${targetParent}/current/media/collectiveaccess"
```

Comment:
Hmmmm, I'd prefer either adding the parameter to the hook, or just put `storage` as a child of `local` rather than a sibling, although as you said that means the mount point is deep in the file system.